change/@ni-nimble-components-9135d7b9-acf1-411e-b8dc-8a73d7b3860f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Note that in your PR description you can say `Resolves #1350` or `Fixes #1350` instead of `See #1350` if you want GitHub to automatically close it when the PR goes in.